docs/en/sql-reference/functions/other-functions.md

```diff
@@ -3688,3 +3720,118 @@ Result:
 ```response
 {'version':'1','serial_number':'2D9071D64530052D48308473922C7ADAFA85D6C5','signature_algo':'sha256WithRSAEncryption','issuer':'/CN=marsnet.local CA','not_before':'May  7 17:01:21 2024 GMT','not_after':'May  7 17:01:21 2025 GMT','subject':'/CN=chnode1','pkey_algo':'rsaEncryption'}
 ```
+
+## lowCardinalityIndices
+
+For each row in the current [block](../../development/architecture.md/#block-block), returns the index of the value in the dictionary of unique values for columns of [LowCardinality](../data-types/lowcardinality.md) type. 
+
+:::note
+The first unique value encountered in the block is enumerated from 1.
+:::
+
+**Syntax**
+
+```sql
+lowCardinalityIndices(col)
+```
+
+**Arguments**
+
+- `col` — a low cardinality column. [LowCardinality](../data-types/lowcardinality.md).
+
+**Returned value**
+
+- returns the index of the value in the dictionary of unique values, for each row in the current block. [UInt64](../data-types/int-uint.md).
+
+**Example**
+
+Query:
+
+```sql
+DROP TABLE IF EXISTS test;
+CREATE TABLE test (s LowCardinality(String)) ENGINE = Memory;
+
+-- insert two blocks of data:
+
+INSERT INTO test VALUES ('one'),('two'),('one'),('one'),('two');
+INSERT INTO test VALUES ('three'),('two'),('one'),('two'),('two'),('three');
+
+SELECT s, lowCardinalityIndices(s) FROM test;
+```
+
+Result:
+
+```response
+   ┌─s───┬─lowCardinalityIndices(s)─┐
+1. │ one │                        1 │
+2. │ two │                        2 │
+3. │ one │                        1 │
+4. │ one │                        1 │
+5. │ two │                        2 │
+   └─────┴──────────────────────────┘
+    ┌─s─────┬─lowCardinalityIndices(s)─┐
+ 6. │ three │                        1 │
+ 7. │ two   │                        2 │
+ 8. │ one   │                        3 │
+ 9. │ two   │                        2 │
+10. │ two   │                        2 │
+11. │ three │                        1 │
+    └───────┴──────────────────────────┘
+```
+## lowCardinalityKeys
+
+For each row in the current [block](../../development/architecture.md/#block-block), returns the keys (unique values) in the dictionary of unique values for columns of [LowCardinality](../data-types/lowcardinality.md) type.
```

Comment:
`lowCardinalityKeys` is a weird function. Anyways, let's fix up the docs:

```
Returns the dictionary values of a [LowCardinality](../data-types/lowcardinality.md) column.
If the block is smaller or larger than the dictionary size, the result will be truncated or extended with default values.
Since LowCardinality have per-part dictionaries, this function may return different dictionary values in different parts.
```